best-practices/styling/index.html

```diff
@@ -1016,17 +1054,180 @@ <h3>Wrapping and breaking text</h3>
 			<section id="page-breaking">
 				<h3>Page breaking and column layout</h3>
 
-				<div class="issue" title="Seeking an example of forcing page breaks"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding page breaks within
+				<div class="issue" title="Need HTML and CSS of forcing page breaks"><span/></div>
+				<aside class="example" title="Forced page break between front matter and body matter"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠁⠰⠊⠧
+⠠⠊⠝⠙⠕⠝⠑⠎⠊⠁⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠚
+⠠⠊⠗⠁⠝⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠁
+⠠⠊⠗⠑⠇⠁⠝⠙⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠉
+⠠⠊⠎⠗⠁⠑⠇⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠙
+⠠⠊⠞⠁⠇⠽⠀⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠐⠀⠼⠛⠋
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠏⠼⠃
+⠀⠀⠀⠀⠀⠀⠀⠠⠺⠕⠗⠇⠙⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀⠠⠥⠎⠁⠛⠑⠀⠀⠀⠀⠀⠀⠼⠉⠙
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠞⠓⠊⠗⠙⠀⠠⠑⠙⠊⠞⠊⠕⠝⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠃⠗⠁⠊⠇⠇⠑⠀⠠⠥⠎⠁⠛⠑⠀⠃⠽⠀⠠⠉⠕⠥⠝⠞⠗⠽⠀⠐⠣⠉⠕⠝⠞⠲⠐⠜⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀					
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠑⠉⠥⠁⠙⠕⠗⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠇⠁⠝⠛⠥⠁⠛⠑⠒⠀⠠⠎⠏⠁⠝⠊⠎⠓⠀⠐⠣⠥⠎⠥⠁⠇⠇⠽⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠥⠝⠉⠕⠝⠞⠗⠁⠉⠞⠑⠙⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀⠠⠎⠞⠁⠝⠙⠁⠗⠙⠎⠀⠠⠎⠑⠞⠀⠃⠽⠒⠀⠀⠀⠀⠀⠀
+⠠⠇⠁⠞⠊⠝⠀⠠⠁⠍⠑⠗⠊⠉⠁⠝⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀⠠⠉⠕⠥⠝⠉⠊⠇⠀⠀⠀⠀
+⠀⠀⠐⠣⠠⠉⠕⠍⠊⠎⠊⠘⠌⠕⠝⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠠⠇⠁⠞⠊⠝⠕⠁⠍⠑⠗⠊⠉⠁⠝⠁⠨⠜⠀⠐⠣⠠⠠⠉⠃⠇⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠇⠁⠞⠊⠝⠀⠠⠁⠍⠑⠗⠊⠉⠁⠝⠀⠠⠃⠇⠊⠝⠙⠀⠠⠥⠝⠊⠕⠝⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠐⠣⠠⠥⠝⠊⠘⠌⠕⠝⠀⠠⠇⠁⠞⠊⠝⠕⠁⠍⠑⠗⠊⠉⠁⠝⠁⠀⠙⠑⠀⠀⠀⠀⠀⠀
+⠀⠀⠉⠊⠑⠛⠕⠎⠐⠜⠀⠐⠣⠠⠠⠥⠇⠁⠉⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠺⠺⠺⠲⠥⠇⠁⠉⠙⠊⠛⠊⠞⠁⠇⠲⠕⠗⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠃⠗⠁⠊⠇⠇⠑⠈⠁⠥⠇⠁⠉⠙⠊⠛⠊⠞⠁⠇⠲⠕⠗⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠉⠕⠝⠎⠑⠚⠕⠀⠠⠊⠃⠑⠗⠕⠁⠍⠑⠗⠊⠉⠁⠝⠕⠀⠙⠑⠇⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀
+⠀⠀⠐⠣⠠⠠⠉⠊⠃⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠙⠞⠁⠈⠁⠕⠝⠉⠑⠲⠑⠎⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠼⠁</code></pre>
+				</aside>
+				<div class="issue" data-number="93" title="Need HTML and CSS of avoiding page breaks within
 				an element"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding page breaks between
+				<aside class="example" title="Spatially arranged horizontal equations"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠀⠀⠀⠀⠀⠠⠙⠕⠥⠛⠇⠁⠎⠀⠠⠁⠙⠁⠍⠎⠒⠀⠠⠁⠀⠠⠏⠊⠕⠝⠑⠑⠗⠀⠕⠋⠀⠀⠀⠀
```

Comment:
Where should the page breaks go in this example?

Reply:
I think I understand the question but please correct me if I'm wrong. You're avoiding page breaks within the box. So including and between the row of dots 2356 and the row of dots 1234.

Reply:
Yes, sorry, quite obvious.

Reply:
No worries! I could have been more clear with a comment or something.